pkg/rebalancer/storage/controller/service/rebalancer.go

```diff
@@ -19,28 +19,513 @@ package service
 
 import (
 	"context"
+	"math"
 	"reflect"
+	"strconv"
+	"sync/atomic"
+	"time"
 
+	"github.com/vdaas/vald/internal/errgroup"
 	"github.com/vdaas/vald/internal/errors"
+	"github.com/vdaas/vald/internal/k8s"
+	"github.com/vdaas/vald/internal/k8s/configmap"
+	"github.com/vdaas/vald/internal/k8s/decoder"
+	"github.com/vdaas/vald/internal/k8s/job"
+	mpod "github.com/vdaas/vald/internal/k8s/metrics/pod"
+	"github.com/vdaas/vald/internal/k8s/pod"
+	"github.com/vdaas/vald/internal/k8s/statefulset"
+	"github.com/vdaas/vald/internal/log"
+	"github.com/vdaas/vald/internal/safety"
+	"github.com/vdaas/vald/pkg/rebalancer/storage/controller/config"
+	"github.com/vdaas/vald/pkg/rebalancer/storage/controller/model"
 )
 
+const (
+	qualifiedName string = "app.kubernetes.io/rebalancer-"
```

Comment:
we don't have kubernetes.io domain.
and this const is used as prefix.
```suggestion
	qualifiedNamePrefix string = "rebalancer.vald.vdaas.org/"
```

Reply:
fixed